CONTRIBUTING.md

```diff
@@ -1,3 +1,99 @@
-Thank you for helping the Streamlink project! Before opening a new ticket,
-please make sure that there's no similar ticket already opened for your
-issue and that it isn't a [known issue](https://streamlink.github.io/players.html#known-issues-and-workarounds).
+# Contributing to Streamlink
+
+Want to get involved? Thanks! There are plenty of ways to help!
+
+
+## Reporting issues
+
+A bug is a *demonstrable problem* that is caused by the **latest version** of the code in the repository. Good bug reports are extremely helpful - thank you!
+
+Please read the following guidelines before you [report an issue][issues]:
+
+1. **See if the issue is already known** — check the list of [known issues][known-issues].
+
+2. **Use the GitHub issue search** — check if the issue has already been reported. If it has been, please comment on the existing issue.
+
+3. **Check if the issue has been fixed** — the latest `master` or development branch may already contain a fix.
+
+4. **Isolate the demonstrable problem** — make sure that the code in the project's repository is *definitely* responsible for the issue.
+
+5. **Format your issue report** — [well formatted texts][mastering-markdown] will make it much easier for developers to understand your report.
+
+Please try to be as detailed as possible in your report too. What is your environment? What steps will reproduce the issue? What would you expect to be the outcome? All these details will help people to assess and fix any potential bugs. The [issue template][issue-template] will aid you in structuring your report when submitting a new issue. If your report includes log outputs, please provide those as text in multi-line code blocks if possible for better readability, thank you!
+
+
+## Feature requests
+
+Feature requests are welcome. But take a moment to find out whether your idea fits with the scope and aims of the project. It's up to *you* to make a strong case to convince the project's developers of the merits of this feature. Please provide as much detail and context as possible.
+
+
+## Plugin requests
+
+Plugin submissions and requests are a great way to improve Streamlink. Requests should be as detailed as possible and dedicated to only a single streaming service. Informations about the service as well as explicit URLs for live streams or recordings are required, so that the requested plugin can be easily implemented. Please be aware that plugins for streaming services that are using DRM protections will not be implemented.
```

Comment:
I don't think we need the `s` on `Informations` here. Minor change here to make it sound less demanding so we don't intimidate people on accident: `recordings are encouraged so that the requested plugin can be easily implemented.`